src/index.js

```diff
@@ -0,0 +1,21 @@
+/*eslint-disable */
+import ReactDOM from 'react-dom/client';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import TodoContainer from './functionBased/components/TodoContainer';
+import About from './functionBased/pages/About';
+import NotMatch from './functionBased/pages/NotMatch';
+import Navbar from './functionBased/components/Navbar';
```

Comment:
Correct the nav bar it should be only visible in Mobile view not in desktop.
![image](https://user-images.githubusercontent.com/13853450/178067650-5b7f2c42-d01d-407e-90a3-c3b67020426f.png)